src/librustc_trans/trans/debuginfo.rs

```diff
@@ -1156,12 +1156,42 @@ pub fn get_cleanup_debug_loc_for_ast_node<'a, 'tcx>(cx: &CrateContext<'a, 'tcx>,
         }
     }
 
-    NodeInfo {
+    NodeIdAndSpan {
```

Comment:
You add it here: https://github.com/rust-lang/rust/pull/20642/files#diff-6186c8f94ed38371395958b0f332cc6aR261